actionpack/lib/action_dispatch/journey/gtg/builder.rb

```diff
@@ -146,16 +136,10 @@ def build_followpos
                 end
               end
             end
-            table
           end
 
           def symbol(edge)
-            case edge
-            when Journey::Nodes::Symbol
-              edge.regexp
-            else
-              edge.left
-            end
+            edge.is_a?(Journey::Nodes::Symbol) ? edge.regexp : edge.left
```

Comment:
> 6. Switch symbol to use a ternary if instead of a case/when

Why? It looks like cosmetic changes.
https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#write-your-code

Reply:
The ternary version is a bit faster and it felt more readable to me, since there's only one case. We can drop this change if the case/when is preferred
```
Warming up --------------------------------------
              symbol   346.933k i/100ms
         fast_symbol   416.123k i/100ms
Calculating -------------------------------------
              symbol      3.354M (±10.3%) i/s -     16.653M in   5.028782s
         fast_symbol      3.964M (± 2.0%) i/s -     19.974M in   5.040992s

Comparison:
         fast_symbol:  3963868.1 i/s
              symbol:  3354284.9 i/s - 1.18x  (± 0.00) slower
```

Reply:
Using a predicate here might make this read even better, plus it also seems to be a bit faster:

```rb
require "benchmark"
require 'benchmark/ips'

x = ActionDispatch::Journey::Nodes::Symbol.new("symbol")

Benchmark.ips do |bm|
  bm.report("predicate") { x.symbol? }
  bm.report("is_a") { x.is_a?(ActionDispatch::Journey::Nodes::Symbol) }
  bm.compare!
end
```

```
Warming up --------------------------------------
           predicate     1.628M i/100ms
                is_a     1.454M i/100ms
Calculating -------------------------------------
           predicate     16.459M (± 1.3%) i/s -     83.015M in   5.044799s
                is_a     14.546M (± 1.3%) i/s -     74.156M in   5.098756s

Comparison:
           predicate: 16458554.7 i/s
                is_a: 14546386.8 i/s - 1.13x  (± 0.00) slower
```

Reply:
Ah, nice!